cli/utils/utils.go

```diff
@@ -95,6 +91,25 @@ func SetViperConfig(cmd *cobra.Command) error {
 	return nil
 }
 
+func getConfigurationPath(cmdUse, configPath, configFileName string) string {
+	var configPathYAML string
+	fileName := "config.yaml" // default file name
+	switch cmdUse {
+	case "init":
+		fileName = "init.yaml"
+	case "reshare":
+		fileName = "reshare.yaml"
+	case "start-operator":
+		// fileName is already set to "config.yaml" by default
+	}
+	if configFileName != "" {
+		configPathYAML = fmt.Sprintf("%s/%s", configPath, configFileName)
+	} else {
+		configPathYAML = fmt.Sprintf("%s/%s", configPath, fileName)
+	}
+	return configPathYAML
+}
```

Comment:
this code is kinda funny, I would for example check `configFileName` before doing the switch block, honestly I would check it out of the function and run this function only if its empty.